docs/redirects.map

```diff
@@ -802,3 +802,5 @@ release-notes.html -> /support/releases.html#release-notes
 support/release-notes.html -> /support/releases.html#release-notes
 support/roadmap.html -> /support/releases.html#roadmap
 app-dev/bindings-ts/testing.html -> /getting-started/testing.html
+2.2.0/deploying.html -> /deploy/generic_ledger
+deploy/index.html -> /deploy/generic_ledger
```

Comment:
```suggestion
deploying.html -> /deploy/generic_ledger.html
deploy/index.html -> /deploy/generic_ledger.html
```

Reply:
I'd actually prefer if all of those redirects were pointing to relative paths, but I guess that's a separate conversation (as I do mean _all of them_, i.e. the entire file).